Library/Homebrew/cmd/cat.rb

```diff
@@ -25,6 +25,7 @@ def cat
     raise "`brew cat` doesn't support multiple arguments" if args.remaining.size > 1
 
     cd HOMEBREW_REPOSITORY
-    safe_system "cat", formulae.first.path, *ARGV.options_only
+    cat_args = Homebrew.args.options_only - CLI::Parser.global_options.values.map(&:first).flatten
```

Comment:
Given this is used more than once it might be nice to have it be a method.

Reply:
Extract out just `CLI::Parser.global_options.values.map(&:first).flatten` or the whole expression?

Reply:
I think the whole expression as a `Homebrew.args` method?

Reply:
`Homebrew.args.passthrough` or something?

Reply:
Addressed.